cmd/juju/application/deploy_test.go

```diff
@@ -138,7 +138,7 @@ func (s *DeploySuite) TestPathWithNoCharmOrBundle(c *gc.C) {
 
 func (s *DeploySuite) TestCharmDir(c *gc.C) {
 	ch := testcharms.Repo.ClonedDirPath(s.CharmsPath, "dummy")
-	err := runDeploy(c, ch, "--series", "trusty")
+	err := runDeploy(c, ch, "--series", "trusty", "--force")
```

Comment:
Wouldn't it just be easier to change our testing charms to be multi-series for the series that we use them for?